client-v2/src/components/FeedSectionHeader.tsx

```diff
@@ -169,16 +175,7 @@ class FeedSectionHeader extends Component<ComponentProps> {
           </FadeTransition>
           <FadeTransition active={!isCurrentFrontsMenuOpen} direction="right">
             {editMode === 'editions' ? (
-              editionsIssue ? (
-                <EditionIssueInfo>
-                  <EditionTitle>
-                    {startCase(editionsIssue.displayName)}
-                  </EditionTitle>
-                  <EditionDate>
-                    {new Date(editionsIssue.issueDate).toDateString()}
-                  </EditionDate>
-                </EditionIssueInfo>
-              ) : null
+              this.renderEditionsActions()
```

Comment:
Is this is a pattern that we're using elsewhere in facia?

Reply:
I don't think so, however pushing Edition specific rendering into its own function makes things easier to read.

Reply:
I agree it makes it easier to read locally but it's mixing styles. Looks like this is done elsewhere with nested components. What's your opinion of this style @jonathonherbert ?